consumer/dragonboard820c/guides/setup-development-environment.md

```diff
@@ -0,0 +1,537 @@
+ # && rm -rf /var/lib/apt/lists/* \
+```
+
+## Install verious other dependencies
+
+```shell
+sudo apt-get install -y python3-pip
+sudo apt-get install -y python3-dev python3-tk python3-numpy
+sudo apt-get install -y python3-scipy
+sudo apt-get install -y python-opencv
+sudo apt-get install -y libopencv-dev
+```
+
+Reboot
+
+```shell
+reboot
+```
+
+## Setup Desktop Environment
+If you installed the developer image you might want to add the Desktop Environment later or you installed the ALIP GUI image, but you want to use a different theme, perhaps another Desktop Manager than sddm.
+
+```shell
+sudo tasksel
+```
+
+This command provides a character based UI.
+Select the following by typing space.
+
+- Debian Desktop Environment
+- All or some indented GUI themes
+- SSH Server
+
+select sddm Desktop Manager when asked.
+  
+Reboot
+
+```shell
+reboot
+```
+
+## Setup linaro user
+
+```shell
+login linaro
+```
+
+Type linaro as a password.
+
+**Add Python to the PYTHONPATH environment variable.**
+
+```shell
+sudo sed -i '$a export PYTHONPATH=/usr/local/lib/python3.6/dist-packages:/usr/local/lib/python2.7/site-packages\n' ~/.bashrc
+```
+
+## Add a new user
+**SSH ACL**
+
+```shell
+sudo groupadd -rf ssh
+sudo usermod -a -G ssh root
+sudo adduser $USER ssh
+```
+
+**I²C ACL**
+
+```shell
+sudo adduser $USER i2c
+```
+
+**logout to root**
+
+```shell
+logout
+```
+
+**Fix /etc/adduser.conf before we add a new user**
+
+```shell
+sed -i 's/#ADD_EXTRA_GROUPS/ADD_EXTRA_GROUPS/' /etc/adduser.conf
+sed -i 's/#EXTRA_GROUPS="dialout cdrom floppy audio video plugdev users"/EXTRA_GROUPS="linaro adm dialout cdrom sudo audio dip video plugdev staff systemd-journal netdev bluetooth ssh pulse-access i2c"/' /etc/adduser.conf
+```
+
+**Add the new user**
+
+```shell
+adduser [new username] --add_extra_groups
+```
+Provide Full name and other info if you wish.
+
+```shell
+login [new username]
+```
+
+Type password
+
+let's test which group the user belongs to
+
+```shell
+groups
+```
+
+You should see this:
+
+```shell
+[new username] adm dialout cdrom sudo audio dip video plugdev staff systemd-journal netdev bluetooth ssh pulse-access i2c linaro
+```
+
+**Add Python to the PYTHONPATH environment variable.**
+
+```shell
+sudo sed -i '$a export PYTHONPATH=/usr/local/lib/python3.6/dist-packages:/usr/local/lib/python2.7/site-packages\n' ~/.bashrc
+```
+
+logout to root
+
+```shell
+logout
+```
+
+## Add SSH keys
+Please follow [these](https://www.ssh.com/ssh/putty/windows/puttygen) instructions to generate SSH private and public keys with PuttyGen on Windows.
+To be able to access the board via SSH properly we copy generated SSH private and public keys.
+
+```shell
+mkdir .ssh
+cd .ssh
+```
+
+**Copy Private Key**
+Fire up your favorite editor (nano) and copy the the content of id_rsa (private key).
+ 
+```shell
+nano id_rsa
+```
+
+**Copy Public Key**
+Copy the the content of id_rsa.pub (public key).
+
+```shell
+nano id_rsa.pub
+```
+
+**Copy Public Key**

```

Comment:
please have a look at ssh-copy-id utility, i would prefer we encourage our users to use it instead of the manual process you describe here.

Reply:
It would be great if it works. It errors out with ```exec request failed on channel 0```.
Until that's resolved we can have these simple couple of steps.

Reply:
I created a new SSH Setup guide